lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -384,41 +384,38 @@
         @include clearfix();
         @include padding(0, 10px, 0, 230px);
 
-        > hgroup {
-          padding: 0;
+        .course-item-info {
+          position: relative;
           width: 100%;
+          padding: 0;
 
-          .university {
-            color: $lighter-base-font-color;
-            font-family: $sans-serif;
-            font-size: 16px;
-            font-weight: 400;
-            margin: 0 0 6px;
-            text-transform: none;
-            letter-spacing: 0;
+          .course-title {
+            display: inline-block;
+            margin-top: -($baseline/2);
+
+            a,
```

Comment:
I would avoid using element based tags here and use class names for this section of styles (and other similar ones)